llama-index-integrations/vector_stores/llama-index-vector-stores-chroma/llama_index/vector_stores/chroma/base.py

```diff
@@ -235,36 +240,46 @@ def from_params(
     def class_name(cls) -> str:
         return "ChromaVectorStore"
 
+    def move_nodes(self, from_index_id: str, to_index_id: str):
+        all_nodes = self._collection.get(where={"__index_id__": from_index_id})
```

Comment:
this would bring every node into memory. Not ideal tbh for large datasets

Reply:
I agree. Not the best solution!
Something like this is probably better: 

        all_metadatas = self._collection.get(where={"__index_id__": from_index_id}, include=[IncludeEnum.metadatas])
        ids = all_metadatas["ids"]
        updated_metadatas = [{**m, "__index_id__": to_index_id} for m in all_metadatas["metadatas"]]
        self._collection.update(ids, metadatas=updated_metadatas)

And maybe there are even better ways of doing it. I haven't looked too deeply into ChromaDB